hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java

```
@@ -326,4 +327,15 @@ public static long retrieveContainerIdFromTarName(String tarName)
           tarName);
     }
   }
+
+  public static long getPendingDeletionBlocks(ContainerData containerData) {
+    if (containerData.getContainerType()
+        .equals(ContainerProtos.ContainerType.KeyValueContainer)) {
+      return ((KeyValueContainerData) containerData)
+          .getNumPendingDeletionBlocks();
+    } else {
+      // If another ContainerType is available later, implement it
+      return 0L;
```

Please throw the exception here.